internal/service/studies/main.go

```diff
@@ -242,8 +247,23 @@ func (s *Service) UpdateStudy(id uuid.UUID, studyData openapi.StudyRequest) erro
 		return types.NewNotFoundError("study not found")
 	}
 	study := studies[0]
-
 	setStudyFromStudyData(&study, studyData)
+
+	studyAdmins, err := s.createStudyAdmins(studyData)
+	if err != nil {
+		return err
+	}
+
+	for _, studyAdminUser := range studyAdmins {
+		studyAdmin := types.StudyAdmin{
+			StudyID: study.ID,
+			UserID:  studyAdminUser.ID,
+		}
+		if err := s.db.Create(&studyAdmin).Error; err != nil {
+			return types.NewErrFromGorm(err, "failed to create study admin")
+		}
```

Comment:
should be idempotent

Reply:
have created https://github.com/ucl-arc-tre/portal/issues/335 since it's a bigger problem and should be handled in a separate PR